src/components/OrdersWidget/index.tsx

```diff
@@ -235,19 +233,22 @@ const OrdersWidget: React.FC<Props> = ({ isWidget = false }) => {
       if (selectedTab === 'fills') return
 
       setClassifiedOrders(curr => {
-        const state = emptyState()
-
         // copy full state
-        Object.keys(curr).forEach(tab => (state[tab] = curr[tab]))
```

Comment:
You know, I was always wondering at the complexity of this code Vs doing what you did. Personally was too scared/lazy to actually do it. Absolute mad man

Reply:
![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fi.imgflip.com%2F30r1af.png&f=1&nofb=1)